src/Ilios/CoreBundle/Entity/User.php

```diff
@@ -140,8 +140,6 @@ class User implements UserInterface
      *
      * @Assert\NotNull()
      * @Assert\Type(type="boolean")
-     *
-     * @JMS\SerializedName("addedViaIlios")
```

Comment:
Yes.  We're not exposing that in the API so this was an extra annotation that left me with the idea the we WERE exposing it.  I took it out to avoid repeat mistakes.